03_Modules/Configuration/jest.config.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import('ts-jest').JestConfigWithTsJest} **/
```

Comment:
Very minor: I found jest also support typescript in their [doc](https://jestjs.io/docs/next/configuration). It would be more consist if we write this config script in ts.

Reply:
fair! will adjust